File: src/main/java/com/flowcommerce/core/domain/member/SellerApply.java
package com.flowcommerce.core.domain.member;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class SellerApply {
    private Long memberId;
    private String sellerName;
    private SellerStatus status;

    private String companyName;
    private String businessNumber;
    private String onlineSalesNumber;
    private String businessType;
    private String companyPhone;
    private String address;

    private String contactName;
    private String contactPhone;
    private String contactEmail;

    private String accountHolder;
    private String bankName;
    private String accountNumber;

    private BigDecimal commissionRate;

    private SettlementBasis settlementBasis;
}

File: src/main/kotlin/com/flowcommerce/core/domain/member/SellerApply.kt
package com.flowcommerce.core.domain.member

import java.math.BigDecimal

data class SellerApply(
    val memberId: Long,
    val sellerName: String,
    val status: SellerStatus,
    val companyName: String,
    val businessNumber: String,
    val onlineSalesNumber: String,
    val businessType: String,
    val companyPhone: String,
    val address: String,
    val contactName: String,
    val contactPhone: String,
    val contactEmail: String,
    val accountHolder: String,
    val bankName: String,
    val accountNumber: String,
    val commissionRate: BigDecimal,
    val settlementBasis: SettlementBasis
)
